docs/guides/actions.rst

```diff
@@ -137,6 +137,38 @@ This is done as follows:
 This lets a plugin extend an existing action without the need to replace the whole action. In the case of the captcha plugin it allows the plugin to provide captcha support in a very loosely coupled way.
 
 
+Actions available in core
+=========================
+
+``action/delete``
+-------------------
+
+If your plugin does not implement any custom logic when deleting an entity, you can use bundled delete action
+
+.. code:: php
+
+   $guid = 123;
+   // You can provide optional forward path as a URL query parameter
+   $forward_url = 'path/to/forward/to';
+   echo elgg_view('output/url', array(
+      'text' => elgg_echo('delete'),
+      'href' => "action/delete?guid=$guid&forward_url=$forward_url",
+      'confirm' => true,
+   ));
+
```

Comment:
Do these get properly syntax highlighted when you test with `grunt`? I think I had to use `.. code-block:: php`.